draft-ietf-mls-architecture.md

```diff
@@ -516,6 +529,10 @@ the rest of the group long enough that the mechanisms above no longer work. This
 can only be solved by sending a ReInit proposal to both groups, possibly with an
 external sender type, and recreating the group to contain all members again.
 
+If the Commit references an unknown proposal, group members may need to solicit
+the Delivery Service or other group members inidividually for the contents of
+the proposal.
```

Comment:
to make sense of the proposal ?

Reply:
To get the proposal message, in the event that it was dropped or hasn't arrived yet